src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -503,6 +511,11 @@ public async Task<List<Instance>> GetInstancesInStateOfInstanceOwner(int instanc
 
             FeedResponse<Instance> feedResponse = await query.ExecuteNextAsync<Instance>();
 
+            if (_settings.CollectMetrics)
```

Comment:
Should we could consider only logging for requests that take longer than a given treshold or does it give value to see metrics for quick queries as well?

Reply:
I guess for comparison it could be good to see if the same query can go quick some time.